python/lsst/ctrl/bps/htcondor/htcondor_service.py

```diff
@@ -1192,6 +1207,7 @@ def _create_detailed_report_from_jobs(wms_workflow_id, jobs):
     """
     _LOG.debug("_create_detailed_report: id = %s, job = %s", wms_workflow_id, jobs[wms_workflow_id])
     dag_job = jobs[wms_workflow_id]
+    task_jobs = {job_id: job_ad for job_id, job_ad in jobs.items() if job_id != wms_workflow_id}
```

Comment:
"task" is an overloaded term especially in PanDA context.  I'd prefer a different name.  Wondering if the code could remove the dag_job from jobs instead of making a new dictionary with everything else?

Reply:
Because `jobs` is a dictionary I guess I was afraid that altering it in-place may have unintended consequences as these changes will be visible outside of the function. But you're right, since it's just a single entry I could just remove it and easily add it back. It will also solve the problem of the variable naming.